### DISCUSSION TITLE
Document upgrade and downgrade processes

### DISCUSSION THREAD

**Discussion by zamazan4ik** [openebs/openebs#3558] [General] [answered]
Hi.

After reading the documentation it is not clear for me, how upgrade and downgrade processes should be performed over the solution. Could you please describe somewhere in the documentation:

- How to upgrade OpenEBS to the newer version (hopefully with no downtime). Now it is not clear, does it provide zero-downtime guarantees or not
- How to downgrade OpenEBS to the older version (hopefully with no downtime)
- Maybe some compatibility policies and notes between versions

Thanks in advance!

**Comment by hrudaya21** [comment, marked as answer]
No OpenEBS engine support downgrade.
Currently Mayastor engine support zero downtime upgrade. Please refer more details from following link.
https://mayastor.gitbook.io/
